modules/aleph/src/Aleph/AlephRequest.php

```diff
@@ -0,0 +1,396 @@
+<?php
+
+namespace Drupal\aleph\Aleph;
+
+/**
+ * Class AlephRequest.
+ *
+ * Entity with setters/getters for Z37 (REQUESTS) in Aleph.
+ *
+ * The Z37 contains requests information.
+ * The Z37 record functions both for requests for
+ * loaned items (waiting requests), and for requests for delivery of available
+ * items. It is not necessarily expected that a library will convert requests
+ * when installing the ALEPH system. In many cases, the library will inform
+ * patrons that there is a system switchover, that outstanding requests are
+ * not retained, and must be re-entered after switchover.
```

Comment:
How a library handles switchover from one system to another is irrelevant for this code, this comment only confuses.